gruntfile.js

```diff
@@ -17,6 +17,9 @@ module.exports = function( grunt ) {
 
 		jshint: {
 			options: {
+				globals: {
+					'CKEDITOR': false
```

Comment:
> If value is false (default), JSHint will consider that variable as read-only.

Good one... but absolutely unintuitive :D.

> As for the quotes, if you really made a question, just because it sounded like a list of "strings", not really properties of an object. :)

Makes sense. @pjasiun will like it ;> We can mention this in our style guide if you think it makes sense there.